tests/ui/eq_op.rs

```diff
@@ -2,7 +2,8 @@
 #[warn(clippy::eq_op)]
 #[allow(clippy::identity_op, clippy::double_parens, clippy::many_single_char_names)]
 #[allow(clippy::no_effect, unused_variables, clippy::unnecessary_operation, clippy::short_circuit_statement)]
-#[warn(clippy::nonminimal_bool)]
+#[allow(clippy::nonminimal_bool)]
```

Comment:
I guess this is tested in another file? Everything else LGTM

Reply:
Yep